source/developers/file-transfer-protocol.rst

```diff
@@ -0,0 +1,228 @@
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. versionadded:: MinIO RELEASE.2023-04-20T17-56-55Z
+
+Overview
+--------
+
+You can use the File Transfer Protocol (FTP) or Secure File Transfer Protocol (SFTP) to interact with the objects on a MinIO deployment.
+
+Supported Protocols
+~~~~~~~~~~~~~~~~~~~
+
+When enabled, MinIO supports FTP access over the following protocols:
+
+- SSH File Transfer Protocol (``SFTP``)
+
+  SFTP is defined by the Internet Engineering Task Force (IETF) as an extension of SSH 2.0.
+  SFTP allows file transfer over SSH for use with :ref:`Transport Layer Security (TLS) <minio-tls>` and virtual private network (VPN) applications.
+
+  Your FTP client must support SFTP.
+
+- File Transfer Protocol over SSL/TLS (``FTPS``)
+  
+  ``FTPS`` allows for encrypted FTP communication with TLS certificates over the standard FTP communication channel.
+  ``FTPS`` should not be confused with ``SFTP``, as ``FTPS`` does not communicate over a Secure Shell (``SSH``).
+
+  Your FTP client must support FTPS.
+
+- File Transfer Protocol (``FTP``)
+  
+  Unencrypted file transfer.
+
+  MinIO does **not** recommend using unencrypted FTP for file transfer.
+
+Supported Commands
+~~~~~~~~~~~~~~~~~~
+
+When enabled, MinIO supports the following ``ftp`` operations:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+   :width: 60%
+
+   * - FTP Client Commands
+     - Supported for MinIO
+
+   * - ``get``
+     - supported
+
+   * - ``put``
+     - supported
+
+   * - ``ls``
+     - supported
+
+   * - ``mkdir``
+     - supported
+
+   * - ``rmdir``
+     - supported
+
+   * - ``delete``
+     - supported
+
+   * - ``append``
+     - not supported
+
+   * - ``rename``
+     - not supported
+
+
+Considerations
+--------------
+
+Versioning
+~~~~~~~~~~
+
+``FTP`` and ``SFTP`` are not version-aware protocols.
+
+- If you use ``FTP`` or ``SFTP`` to get or list objects from a versioned bucket, only the latest version returns.
+- If you ``put`` an object on a versioned bucket with ``FTP`` or ``SFTP`` with the same name as an existing object, MinIO transparently creates a new version of the object, but FTP cannot be aware of this versioning work.
+
+To retrieve or work with any version of an object other than the latest version, you cannot use ``FTP`` or ``SFTP``.
+Use an S3 API Client, such as the :ref:`MinIO Client <minio-client>`.
+
+Authentication and Access
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Authentication to the MinIO deployment through the FTP protocols uses the same identity providers as regular access.
+The following user accounts can access buckets and objects through FTP protocols:
+
+- :ref:`MinIO IDP <minio-internal-idp>` users and their service accounts
+- :ref:`Active Directory/LDAP <minio-external-identity-management-ad-ldap>` users and their service accounts
+- :ref:`OpenID/OIDC <minio-external-identity-management-openid>` service accounts
+
+Rotating credentials **cannot** access buckets or objects over FTP.
```

Comment:
I'm not sure I follow what a rotating credential is. Do you mean STS?

Reply:
Yes, STS. Updated with a link.